tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest3/testDummyStep_MC/vuq_mult.txt

```diff
@@ -0,0 +1 @@
+k_gama = 0.0 ! shut off phase change mass transfer because this is air/water
```

Comment:
Seems to me, you do not need this file. This file will generated by raven, right?